src/components/datepicker/js/calendarMonthBody.js

```diff
@@ -33,8 +33,8 @@
         // of repeated items that are linked, and then those elements have their bindings updated.
         // Since the months are not generated by bindings, we simply regenerate the entire thing
         // when the binding (offset) changes.
-        scope.$watch(function() { return monthBodyCtrl.offset; }, function(offset, oldOffset) {
-          if (offset !== oldOffset) {
+        scope.$watch(function() { return monthBodyCtrl.offset; }, function(offset) {
+          if (angular.isNumber(offset)) {
```

Comment:
This change (here & in other places) doesn't seem to be mentioned in the commit message or issue description.